src/views/BulkEditor.vue

```diff
@@ -0,0 +1,239 @@
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("eCommerce") }}</ion-label>
+            <ion-badge color="danger">{{ $t("no eCommerce selected") }}</ion-badge>
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Run time") }}</ion-label>
+            <ion-label class="ion-text-wrap" @click="() => isOpen = true" slot="end">{{ $t('Select run time') }}</ion-label>
+            <ion-modal  :is-open="isOpen" @didDismiss="() => isOpen = false">
+              <ion-content force-overscroll="false">
+                <ion-datetime
+                  hour-cycle="h12"
+                />
+              </ion-content>
+            </ion-modal>
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Schedule") }}</ion-label>
+            <ion-select interface="popover" :placeholder='$t("Bulk schedule")'>
+              <ion-select-option value="Every 5 minutes">Every 5 minutes</ion-select-option>
+            </ion-select>
+          </ion-item>
+        </ion-card>
+      </section>
+
+      <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+        <ion-fab-button>
+          <ion-icon :icon="iceCreamOutline" />
+        </ion-fab-button>
+      </ion-fab>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import {
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonCardHeader,
+  IonCardTitle,
+  IonCardSubtitle,
+  IonCheckbox,
+  IonContent,
+  IonDatetime,
+  IonFab,
+  IonFabButton,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonItemDivider,
+  IonLabel,
+  IonMenuButton,
+  IonModal,
+  IonNote,
+  IonPage,
+  IonSelect,
+  IonSelectOption,
+  IonTitle,
+  IonToolbar,
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { useStore } from 'vuex';
+import { addOutline, iceCreamOutline, timeOutline, timerOutline } from 'ionicons/icons';
+import { useRouter } from 'vue-router';
+
+export default defineComponent({
+  name: 'InitialLoad',
+  components: {
+    IonBadge,
+    IonButton,
+    IonCard,
+    IonCardContent,
+    IonCardHeader,
+    IonCardTitle,
+    IonCardSubtitle,
+    IonCheckbox,
+    IonContent,
+    IonDatetime,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonItemDivider,
+    IonLabel,
+    IonMenuButton,
+    IonModal,
+    IonNote,
+    IonPage,
+    IonSelect,
+    IonSelectOption,
+    IonTitle,
+    IonToolbar
+  },
+  data(){
+    return {
+      isOpenGlobal: false,
+      isOpen: false
+    }
+  },
+  setup() {
+    const store = useStore();
+    const router = useRouter();
+
+    return {
+      addOutline,
+      iceCreamOutline,
+      store,
+      router,
+      timeOutline,
+      timerOutline
+    }
+  }
+});
+</script>
+
+<style scoped>
+section {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  align-items: start;
+}
+
+ion-button {
+  margin: var(--spacer-base) var(--spacer-xs);
+}
+
+ion-modal {
```

Comment:
this should be globally set right? we are using date time picker through out the app.